packages/genesys-spark-components/src/components/beta/gux-rich-text-editor/gux-rich-text-editor.scss

```diff
@@ -2,15 +2,22 @@
   display: block;
 }
 
+:host([style*='resize']) {
+  overflow: auto;
+  resize: inherit;
+}
+
```

Comment:
@daragh-king-genesys What are your thoughts on this and do you see any downsides to this approach. After looking at various options I think this is okay. My initial thought was to have ,

`:host {
resize:inherit;
overflow:auto;
`
but I dont think this is ideal as it would `inherit` if it was a child of a parent container which I am not sure is desirable.

With the current change it works if they set it on the element itself like 

`gux-rich-text-editor-beta style="resize:both; inline-size:500px;` for example.

Let me know what you think.